index.d.ts

```diff
@@ -20,6 +20,11 @@ export type ThrottledFunction<Argument extends readonly unknown[], ReturnValue>
 		Abort pending executions. All unresolved promises are rejected with a `pThrottle.AbortError` error.
 		*/
 	abort(): void;
+
+	/**
+		The current length of the promise queue.
```

Comment:
I think the docs could be even more explicit about what it is.

> The number of queued items waiting to be executed.

Reply:
Should be sorted on latest commit!